src/PureScript/Backend/Chez/Convert.purs

```diff
@@ -83,6 +86,34 @@ codegenModule { name, bindings, imports, foreign: foreign_ } =
     , body: { definitions, expressions: [] }
     }
 
+definitionIdentifiers :: ChezDefinition -> Array String
+definitionIdentifiers = case _ of
+  DefineValue i _ -> [ i ]
+  DefineCurriedFunction i _ _ -> [ i ]
+  DefineUncurriedFunction i _ _ -> [ i ]
+  DefineRecordType i [ x ] ->
+    [ recordTypeCurriedConstructor i
+    , recordTypePredicate i
+    , recordTypeAccessor i x
+    ]
+  DefineRecordType i fields ->
+    [ recordTypeUncurriedConstructor i
+    , recordTypePredicate i
+    ] <> map (recordTypeAccessor i) fields
+
+resolve :: ModuleName -> Qualified Ident -> String
+resolve _ (Qualified Nothing (Ident i)) = i
+resolve currentModule (Qualified (Just m) (Ident i))
+  | currentModule == m = i
+  | otherwise = coerce m <> "." <> i
```

Comment:
This is entirely optional, but is there a good alternative name for `resolve`? I was thinking `flattenQualified`.

Reply:
I think `flattenQualified` makes sense.